api/elrond_hyper_block_endpoint_test.go

```diff
@@ -0,0 +1,148 @@
+package api
+
+import (
+	"bytes"
+	"encoding/json"
+	"errors"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"strconv"
+	"strings"
+	"testing"
+
+	"github.com/ElrondNetwork/covalent-indexer-go/hyperBlock"
+	"github.com/ElrondNetwork/covalent-indexer-go/testscommon/mock"
+	"github.com/stretchr/testify/require"
+)
+
+func TestElrondHyperBlockEndPoint_GetHyperBlock(t *testing.T) {
```

Comment:
coverage 100% can be easily achieved with 2 more tests, one with nil on constructor and another one with `ReadCloserStub` returning error on `Close`

Reply:
Added tests for 100% coverage